tests/TestHelpers/WebDavHelper.php

```diff
@@ -132,6 +132,10 @@ public static function propfind(
 			}
 			$propertyBody .= "<$namespacePrefix:$property/>";
 		}
+		$folderDepth = "" . $folderDepth;
```

Comment:
I guess this is equivalent to code like:
```suggestion
		$folderDepth = (string) $folderDepth;
```

Reply:
right, that the JS mindset I used here...